.gitignore

```diff
@@ -20,9 +20,13 @@ dist
 /static
 .vscode/
 /build/
-config/standard/.gdrive.*.json
-config/*/backups/*
 api/src/extracted-resources/**/*
 api/build/**/*
 jsdocs
 config/*/.gdrive.*.json
+config/*/.snapshots/*
+config/*/backups/*
+config/*/json_docs/*
+config/*/upload-docs.*
+config/*/users.csv
```

Comment:
Is `users.csv` used to populate an instance with dummy data? If so, may be good to keep that one... or are you including it here to avoid accidentally overwritting it with local changes?

Reply:
The `/config/default/users.csv` is created when you run `csv-to-docs` from the users.csv that is in `/config/default/csv/`  files.

Reply:
Ah, looks good then!